docs/en/02_Developer_Guides/15_Customising_the_Admin_Interface/How_Tos/CMS_Formfield_Help_Text.md

```diff
@@ -47,16 +28,5 @@ TextField::create('MyText', 'My Text Label')
     ->addExtraClass('cms-description-toggle');
 ```
 
-If you want to provide a custom icon for toggling the description, you can do that
-by setting an additional `RightTitle`.
-
-
-```php
-TextField::create('MyText', 'My Text Label')
-    ->setDescription('More <strong>detailed</strong> help')
-    ->addExtraClass('cms-description-toggle')
-    ->setRightTitle('<a class="cms-description-trigger">My custom icon</a>');
```

Comment:
It doesn't work because Right Title gets escaped.
![image](https://user-images.githubusercontent.com/1168676/47538073-de378d00-d925-11e8-801f-5b1695cc8846.png)